### ISSUE TITLE
About distributed parallel training

### ISSUE THREAD

**Issue by 3Der1999** [ShenhanQian/GaussianAvatars#28]
Hi, thanks for your great work! 
I have 4 GPUs and plan to use torchrun to achieve distributed parallel training. Before that, I want to know if this approach will impair the model's capabilities(or, is distributed parallel training necessary?), as the dataset could be loaded at once on a single GPU(`loader_camera_train = DataLoader(scene.getTrainCameras(), batch_size=None, shuffle=True, num_workers=8, pin_memory=True, persistent_workers=True)`). 
Do you have any ideas?
Thanks!

**Comment by ShenhanQian** [comment]
Hi, we didn't explore multi-gpu training because 3D-GS is very fast and memory-efficient. Either training or test can fit on a simple RTX 2080Ti card.

When initiating the dataset, we only read image paths instead of the files to save memory and launching time. The image files are loaded when needed on the fly. Generally speaking, the data loading overhead can be ignored when we use multiple workers for the dataloader.
